bang/forms.py

```diff
@@ -1177,8 +1178,34 @@ def save(self, commit=False):
         if not self.is_creating and instance.member != self.instance.previous_member:
             self.instance.previous_member.force_cache_totals()
 
-        if commit:
-            instance.save()
+        instance.save()
+
+        # Delete existing chibis
+        if not self.is_creating:
+            for imageObject in self.all_chibis:
+                field_name = u'delete_chibi_{}'.format(imageObject.id)
+                field = self.fields.get(field_name)
+                if field and self.cleaned_data[field_name]:
+                    imageObject.delete()
+
+        # Upload new chibis
+        for image in self.cleaned_data['chibis']:
+            if isinstance(image, int):
```

Comment:
Do you know why that happens btw?
It was a hot fix from a while back but I'm not sure I get why images are ints sometimes. 🤔